src/main/java/frc/robot/subsystems/swerve/SwerveModule2020.java

```diff
@@ -0,0 +1,147 @@
+package frc.robot.subsystems.swerve;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+
+import frc.robot.motorcontrol.MotorUtil;
+
+/**
+ * A swerve module with a NEO drive motor and a BAG steer motor, for running swerve modules on 
+ * the 2020 robot.
+ */
+public class SwerveModule2020 {
```

Comment:
wait is this class duplicated

Reply:
The difference between the 2020 swerve module and 2023 is that 2020 uses different motors and needs different PID constants to run. Also 2020 doesn't have the subclass setup in place yet, which we could consider adding.